ts/src/binance.ts

```diff
@@ -9015,9 +8917,14 @@ export default class binance extends Exchange {
         //     }
         //
         const info = this.safeDict (response, 'info', {});
-        const url = this.safeString (info, 'url');
+        let url = this.safeString (info, 'url');
+        if (url === undefined) {
+            // 'info' does not seem to contain 'url' key, however we keep it for backward-compatibility
+            // currently 'response' seems to contain that key
+            url = this.safeString (response, 'url');
```

Comment:
i don't know why `url` was extracted from `info`. just use fetchDepositAddress on any coin and you will see that.